pkg/controller/vitesskeyspace/reconcile_resharding.go

```diff
@@ -120,8 +122,26 @@ func (r *reconcileHandler) reconcileResharding(ctx context.Context) (reconcile.R
 		sort.Strings(errorMsgs)
 		r.setConditionStatus(planetscalev2.VitessKeyspaceReshardingInSync, corev1.ConditionFalse, "Error", fmt.Sprintf("VReplication reported an error: %v", errorMsgs[0]))
 	case planetscalev2.WorkflowCopying:
+		// Aggregate tablet maps for all source shards.
+		sourceRowCount, err := r.shardsRowCount(ctx, workflowStatus.SourceShards)
+		if err != nil {
+			r.recorder.Eventf(r.vtk, corev1.EventTypeWarning, "SourceShardsRowCountFailed", "failed to aggregate row count for source shards: %v", err)
```

Comment:
I think the Reason string is more useful when we use it to group problems into high-level categories, rather than uniquely specifying exactly which case we hit. The Message string tells you the details. How about something like this?

```suggestion
			r.recorder.Eventf(r.vtk, corev1.EventTypeWarning, "CopyProgressUnknown", "failed to aggregate row count for source shards: %v", err)
```

Reply:
Sounds good to me

Reply:
Done.